rq_dashboard/templates/rq_dashboard/base.html

```diff
@@ -33,6 +33,13 @@ <h3>Do you really want to <span id="confirmation-modal-action"></span>?</h3>
   </div>
 </div>
 
+<footer id="version-footer" class="footer-navbar-fixed-bottom">
```

Comment:
There is no need to make footer sticky: there is no universally useful info in It, so there is no need to waste space on user's screen.